src/ui/screens/private-channels/PrivateChannelsListView.tsx

```diff
@@ -7,15 +7,16 @@ import { ContactGrid } from '../contact/ContactGrid';
 import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
 import { ComponentColors } from '../../../styles';
 import { WideButton } from '../../buttons/WideButton';
+import { Feed } from '../../../models/Feed';
 
 export interface DispatchProps {
-    onPressChannel: (feed: ContactFeed) => void;
+    onPressChannel: (feed: ContactFeed | Feed) => void;
     onAddChannel: () => void;
 }
 
 export interface FeedSection {
     title?: string;
-    data: ContactFeed[];
+    data: Array<Feed | ContactFeed>;
```

Comment:
Why this definition has changed? On the PrivateChannelListView we can only display ContactFeeds, not ordinary Feeds.

Reply:
I was unsure about both of these occurrences because of the sharing with self usecase